packages/ag-charts-community/src/chart/interaction/tooltipManager.ts

```diff
@@ -106,4 +110,46 @@ export class TooltipManager {
 
         this.appliedState = { content: contentToApply, meta: metaToApply };
     }
+
+    public static makeTooltipMeta(
```

Comment:
Should this static method be moved into its own file (e.g. _tooltipUtils.ts_)?

The new imports are creating a lot of warnings in `nx run ag-charts-community:lint-depcruise` (although this should be okay because it's only `import type` statements).

Reply:
So I don't think moving this method to a new file will help with the circular deps; the problem is the dependency back to `SeriesNodeDatum` and `ErrorBoundSeriesNodeDatum ` - we need to either not use these, or break them into their own minimally dependent modules?

I'd prefer if we resolve this now if straightforward.

Reply:
Okay. I think that moving `SeriesNodeDatum` and `ErrorBoundSeriesNodeDatum` is the preferred option. So I'll see whether that's viable.

Not using them (i.e. redefining the type contract in _tooltipManager.ts_) will be our Plan B.

Reply:
I've moved these types into _seriesTypes.ts_ just below the `ISeries` interface. This seems like a decent solution. Please let me know your thoughts.